src/Builder/Type/Encode.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Builder\Type;
+
+/**
+ * Defines how to encode a stage or an operator into BSON.
+ *
+ * @see BuilderEncoder
```

Comment:
This exists in a different namespace: MongoDB\Builder. Do you need a `use` statement?